ext/pcntl/pcntl.c

```diff
@@ -892,9 +892,12 @@ PHP_FUNCTION(pcntl_signal)
 		RETURN_THROWS();
 	}
 
-	if (signo < 1 || signo >= NSIG) {
-		php_error_docref(NULL, E_WARNING, "Invalid signal");
-		RETURN_FALSE;
+	if (signo < 1) {
+		zend_argument_value_error(1, "must be greater than or equal to 1");
+		RETURN_THROWS();
+	} else if (signo >= NSIG) {
```

Comment:
Style nit: Avoid else if previous branch returned.

Reply:
Agreed! :) it must have been an overlook